NeoML/src/TraditionalML/SvmBinaryModel.h

```diff
@@ -40,13 +40,13 @@ class CSvmBinaryModel : public ISvmBinaryModel {
 	double GetFreeTerm() const override { return freeTerm; }
 
 protected:
-	~CSvmBinaryModel() override = default; // delete prohibited
```

Comment:
_minor_ unlike `ISvmBinaryModel` this class isn't public. So, this problem with `typeid` is irrelevant (this class can't be created from other module).

Reply:
fixed